### ISSUE TITLE
Enable #brave-split-view

### ISSUE THREAD

**Issue by rebron** [brave/brave-browser#38710]
### Description

Enable brave://flags/#brave-split-view by default.

### Steps to reproduce

1. Go to brave://flags/#brave-split-view
2. Default status should be enabled

### Actual result

n/a

### Expected result

Split view should be enabled by default. 

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

1.75.x

### Channel information

- [ ] release (stable)
- [ ] beta
- [X] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

**Comment by solrevdev** [comment]
This is awesome. When would this be a candidate for going into the normal stable version?

**Comment by MadhaviSeelam** [comment]
Verification `PASSED` using
```
Brave | 1.76.52 Chromium: 133.0.6943.54 (Official Build) beta (64-bit)
-- | --
Revision | 5a91fa2d5c8a9f8d2f1bafa26281c840921fc1c2
OS | Windows 11 Version 24H2 (Build 26100.2894)
```

1. Installed 1.76.52
2. launched Brave
3. opened `brave://flags/#brave-split-view` in a new tab
4. confirmed `Default (Enabled`) is shown
5. opened a new tab context menu
6. confirmed `New split view` option is shown 
7. selected `New split view` option
8. confirmed split view is shown
9. enabled vertical tab view via context view
10. confirmed split view is shown as expected


step 4 | step 6 | step 8 | step 10
----- | ------ | ------ | ----
![Image](https://github.com/user-attachments/assets/fde82e0b-9cc2-41e7-8693-224c0af5a5ac)|![Image](https://github.com/user-attachments/assets/d0f0f060-740a-4e7c-9e7c-a35fb31b98f9)|![Image](https://github.com/user-attachments/assets/cdf65ed1-a560-405f-94bf-e6a3e2fe3633)|![Image](https://github.com/user-attachments/assets/04eb06e8-5634-4380-ad71-c1c2bb98a4aa)

**Comment by Arcitec** [comment]
@solrevdev 

> This is awesome. When would this be a candidate for going into the normal stable version?

Sounds like it will be in 1.76.x which comes out on March 4th:

https://github.com/brave/brave-browser/wiki/Brave-Release-Schedule#release-channel-dates

And seems like it will be enabled by default. It looks so useful!

**Comment by Arcitec** [comment]
@rebron Hi! The feature is here now. Was a decision made to disable it by default?

I am on Fedora Linux with your native repo's RPM package:

`Version 1.76.73 Chromium: 134.0.6998.45 (Official Build) (64-bit)`

I checked: `brave://flags/#brave-split-view`

Result: `Default (Disabled)`

![Image](https://github.com/user-attachments/assets/648eb2a3-066c-4310-82df-ce9c685e470f)

**Comment by joshjohanning** [comment]
> [@rebron](https://github.com/rebron) Hi! The feature is here now. Was a decision made to disable it by default?
> 
> I am on Fedora Linux with your native repo's RPM package:
> 
> `Version 1.76.73 Chromium: 134.0.6998.45 (Official Build) (64-bit)`
> 
> I checked: `brave://flags/#brave-split-view`
> 
> Result: `Default (Disabled)`
> 
> ![Image](https://github.com/user-attachments/assets/648eb2a3-066c-4310-82df-ce9c685e470f)

Nice! Just enabled it via the flag 😄

**Comment by Arcitec** [comment]
> Nice! Just enabled it via the flag 😄

Yeah, but this ticket was about enabling it by default. It's now supposed to say `Default (Enabled)` for everyone. So I wonder if a mistake was made in the Linux packaging where Brave forgot to enable it for Linux users.

**Comment by jilherme** [comment]
> Yeah, but this ticket was about enabling it by default. It's now supposed to say `Default (Enabled)` for everyone. So I wonder if a mistake was made in the Linux packaging where Brave forgot to enable it for Linux users.

Version 1.76.74 Chromium: 134.0.6998.89 (Official Build) (64-bit) 
Windows 11 
here is not enabled by default too

**Comment by joshjohanning** [comment]
macOS same
